packages/logger/src/logBuffer.ts

```diff
@@ -0,0 +1,92 @@
+import { buffer } from 'node:stream/consumers';
+import { it } from 'node:test';
+
+export class SizedItem<V> {
+  public value: V;
+  public logLevel: number;
+  public byteSize: number;
+
+  constructor(value: V, logLevel: number) {
+    this.value = value;
```

Comment:
At least for now, even if we made it generic, let's enforce that `value` should be a string at runtime and if it's not, throw an error. We'll use `import { isString } from '@aws-lambda-powertools/commons/typeutils';` to test it.

In the constructor, as first thing, we do this:

```ts
if (!isString(value)) {
  throw new Error('Value should be a string');
}
```

and then remove the `JSON.stringify()` from the line below.